src/azure-cli/azure/cli/command_modules/dms/_help.py

```diff
@@ -287,7 +287,9 @@
                     // optional, migrates the enumerated events
                     'selected_events': [
                         'sourceSchema1.nightly_maintenance'
-                    ]
+                    ],
+                    // Optional. If true, the database will be selected for schema migration.
+                    "select_database_for_schema_migration": "true|false"
```

Comment:
Maybe we should use the opportunity and also mention that MySQL migration supports OnlineMigration and ReplicateChanges above, at the line #194?